checkerScripts.js

```diff
@@ -87,6 +86,105 @@ function createCheckersBoard(){
 
 		}
 	}
+}
+
+
+
+var board = [[-1, 0,-1, 0,-1, 0,-1, 0],
+		     [ 0,-1, 0,-1, 0,-1, 0,-1],
+			 [-1, 0,-1, 0,-1, 0,-1, 0],
+			 [ 0, 0, 0, 0, 0, 0, 0, 0],
+			 [ 0, 0, 0, 0, 0, 0, 0, 0],
+			 [ 0, 1, 0, 1, 0, 1, 0, 1],
+			 [ 1, 0, 1, 0, 1, 0, 1, 0],
+			 [ 0, 1, 0, 1, 0, 1, 0, 1]];
+
+var isSelected = false;
+var selectedPiece = null;
+var rigthPiecePlace = null;
+var leftPiecePlace = null;
+var selectedPieceColor = null;
+
+function getPieceMovement(){
+	
+	var coordX = parseInt($(this).attr('name')[0]);
+	var coordY = parseInt($(this).attr('name')[1]);
+	
+	
+	if(board[coordX][coordY]==1){
+		var topLeftBoxIndex = ((coordX > 0) && (coordY > 0)) ? (coordX-1)+''+(coordY-1) : null;
+		var topRightBoxIndex = (coordX > 0) ? (coordX-1)+''+(coordY+1) : null;
+		selectedPiece = coordX+''+coordY;
+
+		coloredPiecePlaces(topLeftBoxIndex, topRightBoxIndex);
+		selectedPieceColor = 1;
+
+	}else if(board[coordX][coordY]==-1){
+		var bottomLeftBoxIndex = ((coordX > 0) && (coordY > 0)) ? (coordX+1)+''+(coordY-1) : null;
+		var bottomRightBoxIndex = (coordX > 0) ? (coordX+1)+''+(coordY+1) : null;
+		selectedPiece = coordX+''+coordY;
+
+		coloredPiecePlaces(bottomLeftBoxIndex, bottomRightBoxIndex);
+		selectedPieceColor = -1;
```

Comment:
You are repeating the logic here, I think you can put this in a function and pass the operator **+** or **-** as a parameter.